docs/options.md

```diff
@@ -366,6 +369,7 @@ Default: `auto`
 |---|---|---|---|---|
 | Linux / Intel | `x86_64` | `x86_64` `i686` | `x86_64` | `i686` |
 | Windows / Intel | `AMD64` | `AMD64` `x86` | `AMD64` | `x86` |
+| Windows / ARM64 | `ARM64` | `ARM64` `AMD64` `x86` | `ARM64` `AMD64` | `x86` |
```

Comment:
I see the auto archs are reproduced here also.

Reply:
Consistency 😉